app/gunicorn.conf.py

```diff
@@ -7,11 +7,10 @@
 
 For more information, see https://docs.gunicorn.org/en/stable/configure.html
 """
-import os
 
-from src.app_config import AppConfig
+from src import shared
 
-app_config = AppConfig()
+app_config = shared.get_app_config()
```

Comment:
Modified to use shared.get_app_config() consistently

Reply:
Good idea -- looks like [there is a way](https://stackoverflow.com/questions/8212053/private-constructor-in-python) but it's not simple